roles/maas_postgres/tasks/teardown.yaml

```diff
@@ -9,16 +9,10 @@
 
 - name: "Uninstall Postgres"
   ansible.builtin.apt:
-    name: "{{ maas_postgres_deb_name | default('postgresql-' ~ maas_postgres_version_number) }}"
+    name:
+      - "{{ maas_postgres_deb_name | default('postgresql-' ~ maas_postgres_version_number) }}"
+      - "acl"
+      - "python3-psycopg2"
     state: absent
     purge: true
-
-- name: "Uninstall acl"
-  ansible.builtin.apt:
-    name: acl
-    state: absent
-
-- name: "Uninstall psycopg2"
-  ansible.builtin.apt:
-    name: "python3-psycopg2"
-    state: absent
+    update_cache: true
```

Comment:
shouldn't need to `apt-get update` for removing - this just operates on the local system

Reply:
Fair point, I think I added update_cache to a few removal tasks